### PR TITLE
Second tutorial

### PR DESCRIPTION
**Pull request by L2oukmane**

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,4 +5,11 @@ VALUES
 ('Gabumon','2018-11-15',2,8,true), 
 ('Pikachu','2021-01-07',1,15.04,false),
 ('devimon','2017-05-12',5,11,true),
+('Charmander','2020-03-08',0,11,false),
+('Plantmon' , '2021-11-15',2,5.7,true),
+('Squirtle','1993-04-02',3,12.13,false),
```
Comment (alexander16108) [inline data.sql]: your data are incorrect please check them, correct them properly and check again for a re-review

**Review by alexander16108 — APPROVED** [review]
Hi @L2oukmane , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (KDlamini) [reply data.sql]: Not fullfield.

File: data.sql [hunk 2]
```diff
@@ -5,4 +5,11 @@ VALUES
 ('Gabumon','2018-11-15',2,8,true), 
 ('Pikachu','2021-01-07',1,15.04,false),
 ('devimon','2017-05-12',5,11,true),
+('Charmander','2020-02-08',0,11,false),
+('Plantmon' , '2021-11-15',2,5.7,true),
+('Squirtle','1993-04-02',3,12.13,false),
+('Angemon','2005-06-12',1,45,true),
+('Boarmon','2005-06-07',7,20.4,true),
+('Blossom','1998,10-13',3,17,true),
+('Ditto','2022-05-14',4,22,true)
```
Comment (KDlamini) [inline data.sql]: Some of the weights are supposed to be negative as per project requirements.
e.g 

- `Charmander` is supposed to have a weight of `-11.0`
-`Plantmon` is supposed to have a weight of `-5.7`
- `Squirtle` is supposed to have a weight of `-12.13` and so forth...

Kindly correct all your weights and make sure that all negative weights are inserted.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @L2oukmane 🙋‍♀️,  

You have done an incredible job implementing some of the changes requested by the previous reviewer. 👏🏾
However, there are still some changes that aren't addressed yet. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @student_name, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

- [ ] [OPTIONAL] I'll suggest you add a meaningful description for your PR. For now, your PR description is empty. kindly, add a description for it.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**